templates/vue/src/utils/constants.js

```diff
@@ -29,4 +29,12 @@ export const licenses = {
   },
 }
 
+export const nodeStatus = {
+  PUBLISH: "publish",
+  DRAFT: "draft",
+  SUBMIT: "submitted",
+  REJECT: "reject",
+  ACCEPT: "accept",
```

Comment:
To keep consistent with everywhere else, can we keep these as `rejected` and `accepted`?

Reply:
These were the strings used in #680, so I kept it as is in case there are any nodes in prod with these statusses.

Reply:
That released was skipped as per Steven's request, so you can safely change these. Thanks for thinking of that though!